google key

```diff
@@ -0,0 +1,3 @@
+start
+626749246571-vlg7ga8tnq9abc0fddnbppepkm4l8nbp.apps.googleusercontent.com
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Google Oauth Key' was found.
Severity: Critical

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |

 <!-- fd44f1fe-db1c-47e1-bda1-3c871611b3f8 -->